src/classes/Queue.js

```diff
@@ -0,0 +1,20 @@
+import { DoubleLinkedList } from "./DoubleLinkedList";
+import { Node } from "./Node";
+class Queue {
+    constructor() {
+        this._list = new DoubleLinkedList();
+    }
+
+    add(data) {
+        let node = new Node(data);
+        this._list.push(node);
+    }
```

Comment:
if you want to keep this approach: `const node`, not `let node`

If you agree with the other suggestions:

```suggestion
    add(data) {
        this._list.push(data);
    }
```